### ISSUE TITLE
Broken links in labsjdk-ce-11.0.9-jvmci-20.3-b03

### ISSUE THREAD

**Issue by zakkak** [graalvm/labs-openjdk-11#11]
Hello, 

labsjdk-ce-11.0.9-jvmci-20.3-b03 appears to include several broken symlinks under `legal/`:
```
$ find /opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03 -xtype l 
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.rmic/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.rmic/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.rmic/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.crypto.ec/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.crypto.ec/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.crypto.ec/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.localedata/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.localedata/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.localedata/cldr.md
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.localedata/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jstatd/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jstatd/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jstatd/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.scripting/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.scripting/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.scripting/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.instrument/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.instrument/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.instrument/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.security.auth/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.security.auth/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.security.auth/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.editpad/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.editpad/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.editpad/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.ldap/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.ldap/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.ldap/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.dns/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.dns/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.dns/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.xml/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.xml/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.xml/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.charsets/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.charsets/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.charsets/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.se/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.se/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.se/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.unsupported/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.unsupported/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.unsupported/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.scripting.nashorn/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.scripting.nashorn/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.scripting.nashorn/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jartool/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jartool/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jartool/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.xml.dom/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.xml.dom/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.xml.dom/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.smartcardio/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.smartcardio/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.smartcardio/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.sctp/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.sctp/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.sctp/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.desktop/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.desktop/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.desktop/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.accessibility/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.accessibility/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.accessibility/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jcmd/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jcmd/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jcmd/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.javadoc/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.javadoc/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.javadoc/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdi/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdi/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdi/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jsobject/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jsobject/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jsobject/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.jvmstat/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.jvmstat/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.jvmstat/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.datatransfer/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.datatransfer/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.datatransfer/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.opt/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.opt/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.opt/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jlink/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jlink/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jlink/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.scripting.nashorn.shell/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.scripting.nashorn.shell/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.scripting.nashorn.shell/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jfr/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jfr/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jfr/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdeps/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdeps/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdeps/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.ci/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.ci/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.ci/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.hotspot.agent/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.hotspot.agent/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.hotspot.agent/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jconsole/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jconsole/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jconsole/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.prefs/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.prefs/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.prefs/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.crypto.cryptoki/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.crypto.cryptoki/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.crypto.cryptoki/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.pack/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.pack/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.pack/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.unsupported.desktop/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.unsupported.desktop/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.unsupported.desktop/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.ed/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.ed/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.ed/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.naming/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.naming/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.naming/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.net.http/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.net.http/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.net.http/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.compiler/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.compiler/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.compiler/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.net/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.net/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.net/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.sql.rowset/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.sql.rowset/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.sql.rowset/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.sql/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.sql/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.sql/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.le/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.le/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.le/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management.agent/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management.agent/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management.agent/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.compiler/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.compiler/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.compiler/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.attach/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.attach/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.attach/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.compiler/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.compiler/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.compiler/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.httpserver/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.httpserver/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.httpserver/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jshell/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jshell/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jshell/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.logging/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.logging/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.logging/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdwp.agent/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdwp.agent/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.jdwp.agent/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management.jfr/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management.jfr/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.management.jfr/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.transaction.xa/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.transaction.xa/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.transaction.xa/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.security.jgss/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.security.jgss/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.security.jgss/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.security.sasl/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.security.sasl/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.security.sasl/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.security.jgss/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.security.jgss/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.security.jgss/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.management/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.management/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.management/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.dynalink/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.dynalink/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.dynalink/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.zipfs/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.zipfs/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.zipfs/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.management.rmi/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.management.rmi/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.management.rmi/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.rmi/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.rmi/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.rmi/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.compiler.management/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.compiler.management/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.internal.vm.compiler.management/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.rmi/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.rmi/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/jdk.naming.rmi/ASSEMBLY_EXCEPTION
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.xml.crypto/ADDITIONAL_LICENSE_INFO
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.xml.crypto/LICENSE
/opt/jvms/labsjdk-ce-11.0.9-jvmci-20.3-b03/legal/java.xml.crypto/ASSEMBLY_EXCEPTION
```

**Comment by zakkak** [comment]
The issue persists in labsjdk-ce-11-jvmci-21.0-b03

**Comment by zakkak** [comment]
Seems to be fixed in 21.1 and 21.2 builds
